src/Till.js

```diff
@@ -0,0 +1,50 @@
+var Till = function() {
+  this.detailsList = hipsterCoffeeDetails;
+  this.orderedItems = [];
+  this.subTotal = 0;
+  this.totalCalculator = new TotalCalculator(this.detailsList, this.orderedItems, this.subTotal);
+};
+
+Till.prototype.retrievePrice = function(item) {
+  return this.detailsList[0].prices[0][item];
+};
+
+Till.prototype.addItem = function(item) {
+  this.subTotal += this.retrievePrice(item);
+  this.orderedItems.push(item);
+  return item;
+};
+
+Till.prototype.calculateSubTotal = function() {
+  return this.subTotal;
+};
+
+Till.prototype.calculateTax = function(totalOrderValue) {
+  var tax = totalOrderValue * 0.0864;
+  return Math.round(tax * 100) / 100;
+};
+
+Till.prototype.printOrder = function() {
```

Comment:
This function is not in the right place - if it does belong in another object, it should call on your till and tell it to give it the orderedItems, then print based on that. If it's not being used yet, delete it!